src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,24 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory sessionFactory = buildSessionFactory();
+
+    private HibernateUtil() {
+
+    }
+
+    private static SessionFactory buildSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new ExceptionInInitializerError("Initial SessionFactory creation failed.");
+        }
```

Comment:
this `try-catch` makes no sense

`ExceptionInInitializerError` is an error,  not exception - you shouldn't throw it

if something goes wrong during `buildSessionFactory` method - you'll get `ExceptionInInitializerError`, therefore catching exception makes no sense

simply remove try-catch from this method

Reply:
Yeah. At first I wanted to have the custom message here, but now I see how generic this is. Gotta remove then